FAQ.md

```diff
@@ -5,22 +5,19 @@ title: Frequently Asked Questions
 site_side: true
 tags: security-qualitative-metrics, background, methodology
 project: true
-level: 4
 type: document
 pitch: The OWASP Security Qualitative Metrics is the most detailed list of metrics which evaluate security level of web projects. It shows the level of coverage of OWASP ASVS. 
 ---
 
+# ![Project Logo](assets/images/logo3_small.png) OWASP Security Qualitative Metrics
 
-# ![Project Logo](images/logo3_small.png) OWASP Security Qualitative Metrics  
-# [![OWASP Flagship](https://img.shields.io/badge/owasp-flagship-blue.svg)](https://owasp.org/projects/)
- [![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/OWASP/www-project-security-qualitative-metrics/blob/master/LICENSE)
+[![OWASP Incubator](https://img.shields.io/badge/owasp-incubator-blue.svg)](https://owasp.org/projects/)
+[![GitHub License](https://img.shields.io/github/license/OWASP/www-project-security-qualitative-metrics)](https://github.com/OWASP/www-project-security-qualitative-metrics/blob/master/LICENSE)
+[![GitHub Release](https://img.shields.io/github/v/release/OWASP/www-project-security-qualitative-metrics)](https://github.com/OWASP/www-project-security-qualitative-metrics/releases)
 
- [![GitHub release](https://img.shields.io/github/release/Naereen/StrapDown.js.svg)](https://github.com/OWASP/www-project-security-qualitative-metrics/releases)
+# F.A.Q. 
 
-
-
-
-# F.A.Q. [![Gitter](https://gitter.im/owasp-www-project-security-qualitative-metrics/community)]
```

Comment:
It doesn't make sense yyo have a link in a heading

Reply:
I have seen some samples like this but it is not important. Oiginally it wasn't in the heading. Actually, this site didn't have any problem visually. IT was already checked. I do not understand why these updates come now.

Reply:
Because I went through the list of flagship projects and notice this one I didn't recognize. I found all these thu us good be fixed. So I submitted the changes for you to merge.